test/tests/globalparams/global_param/tests

```diff
@@ -1,12 +1,28 @@
 [Tests]
   design = 'Parser.md'
-  issues = '#437'
+  issues = '#437 #8761'
 
-  [./global_param_test]
+  [global_param_test]
     type = 'Exodiff'
     input = 'global_param_test.i'
     exodiff = 'out.e'
 
     requirement = 'The system shall include an input file syntax that supports defining global options.'
-  [../]
+  []
+  
+  [test_suppress_ignore]
+    type = 'RunException'
+    input = 'suppresscheck.i'
+    cli_args = "Adaptivity/Markers/boundary/use_displaced_mesh=true"
+    expect_err = "The parameter 'Adaptivity/Markers/boundary/use_displaced_mesh' is a private parameter and should not be used in an input file."
+    mesh_mode = REPLICATED
+    requirement = 'The system shall error on locally suppressed parameters even when a global parameter is being ignored.'
```

Comment:
```suggestion
    requirement = 'The system shall error if a private / suppressed parameter is set in the input file.'
```
the global parameter is irrelevant here, local parameters always take precedence